jenkins/dockerized-e2e-runner.sh

```diff
@@ -30,7 +30,7 @@ mkdir -p "${HOST_ARTIFACTS_DIR}"
 : ${JENKINS_GCE_SSH_PRIVATE_KEY_FILE:='/var/lib/jenkins/gce_keys/google_compute_engine'}
 : ${JENKINS_GCE_SSH_PUBLIC_KEY_FILE:='/var/lib/jenkins/gce_keys/google_compute_engine.pub'}
 
-KUBEKINS_E2E_IMAGE_TAG='v20161004-f3120c3'
+KUBEKINS_E2E_IMAGE_TAG='v20161007-34648c8'
```

Comment:
in my ideal world we'd use the hashes to automatically compute the diffs between images when we change.

in the real world, as you note, we pull artifacts from other repos, so the sha doesn't say much.

also, unless you wait until your changes have been merged upstream, and then only build an image at an upstream commit, this sha will likely be meaningless. and doing that is admittedly a real pain.

hopefully soon we can make CI build these images for us. it'd be a lot easier to track changes then, since everything would be automated.